packages/react-components/src/components/StreamMedia.tsx

```diff
@@ -17,6 +23,10 @@ export interface StreamMediaProps {
   videoStreamElement: HTMLElement | null;
   /** Decides whether to mirror the video or not. */
   isMirrored?: boolean;
+  /** Weather the remote stream is receiving data */
+  isReceiving?: boolean;
```

Comment:
to prevent breaking changes this needs to default to `true` (another reason I would recommend renaming to something like `loadingState`)